documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -499,13 +503,9 @@ The following table shows the default heap values.
 
 |===
 
-Heap configuration values should make the best use of a container's xref:con-common-configuration-resources-reference[memory request] without exceeding it.
-
-* If a memory request or limit is specified, a JVM's minimum and maximum memory is set to a value corresponding to the limit.
-* If a memory request or limit is not specified, a JVM's minimum memory is set to `128M`.
-The JVM's maximum memory is not defined to allow the memory to increase as needed. This is ideal for single node environments in test and development.
-
-Total JVM memory usage can be a lot more than the maximum heap size.
```

Comment:
I think thi is important nd should be kept.

Reply:
We still have this. It's just no longer a bullet.